Documentation/BuildInstructions.md

```diff
@@ -47,282 +39,62 @@ sudo sed -i '$d' /etc/apt/sources.list
-- osxfuse, e2fsprogs, m4, autoconf, automake, libtool and `BuildFuseExt2.sh` are needed if you want to build the root filesystem disk image natively on macOS. This allows mounting an EXT2 fs and also installs commands like `mke2fs` that are not available on stock macOS.
-- Installing osxfuse for the first time requires enabling its system extension in System Preferences and then restarting your machine. The output from installing osxfuse with brew says this, but it's easy to miss.
-- bash is needed because the default version installed on macOS doesn't support globstar
-- If you install some commercial EXT2 macOS fs handler instead of osxfuse and fuse-ext2, you will need to `brew install e2fsprogs` to obtain `mke2fs` anyway.
-- As of 2020-08-06, you might need to tell the build system about your newer host compiler. Once you've built the toolchain, navigate to `Build/i686/`, `rm -rf *`, then run `cmake ../.. -G Ninja -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10`, then continue with `ninja install` as usual.
-- If you are on macOS Big Sur, you will need to manually enable QEMU's acceleration before running Serenity, by creating a new file called `entitlements.xml` in the `Build/` folder, with the content below, and then run the command: `codesign -s - --entitlements entitlements.xml --force /usr/local/bin/qemu-system-x86_64`; otherwise the run command will fail.
-
-<details>
-<summary>Content for 'entitlements.xml'.</summary>
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
-	"http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-    <key>com.apple.security.hypervisor</key>
-    <true/>
-</dict>
-</plist>
-```
+### Other systems
 
-</details>
+There is also documentation for installing the build prerequisites for some less commonly used systems:
 
-### OpenBSD prerequisites
-
-```console
-$ doas pkg_add bash cmake g++ gcc git gmake gmp ninja ccache rsync
-```
-
-To use `ninja image` and `ninja run`, you'll need Qemu and other utilities:
-
-```console
-$ doas pkg_add coreutils qemu sudo
-```
-
-### FreeBSD prerequisites
-
-```console
-$ pkg install bash coreutils git gmake ninja sudo gmp mpc mpfr ccache rsync
-```
-
-### Windows prerequisites
-
-For Windows, you will require Windows Subsystem for Linux 2 (WSL2). [Follow the WSL2 instructions here.](https://github.com/SerenityOS/serenity/blob/master/Documentation/NotesOnWSL.md)
-Do note the ```Hardware acceleration``` and ```Note on filesystems``` sections, otherwise performance will be terrible.
-Once you have installed a distro for WSL2, follow the Linux prerequisites above for the distro you installed, then continue as normal.
-
-You may also want to install [ninja](https://github.com/ninja-build/ninja/releases)
+* [Other Linux distributions and *NIX systems](BuildInstructionsOther.md)
+* [Windows](BuildInstructionsWindows.md)
+* [macOS](BuildInstructionsMacOS.md)
```

Comment:
Feels like we could use a survey of what systems people are using for this list segregation :p

I suspect that Windows and macOS are more common that Fedora, but I could be wrong ;)